tools/build-stm-latest.sh

```diff
@@ -39,4 +39,5 @@ do_build pybv3-network PYBV3 MICROPY_PY_WIZNET5K=1 MICROPY_PY_CC3K=1
 do_build pybv10 PYBV10
 do_build pybv10-network PYBV10 MICROPY_PY_WIZNET5K=1 MICROPY_PY_CC3K=1
 do_build stm32f4disc STM32F4DISC
+do_build stm32f429disc STM32F429DISC
```

Comment:
When you goto update your pull request, you can do git commit --amend and a git push -f and it will stay as one commit. I generally post a note to say that I've done that because github won't tell anybody.

The other option is to add more commits and then squash them all back into one using git rebase -i and then do a git push -f